diff --git a/tensorflow/lite/tools/cmake/modules/xnnpack.cmake b/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
index 937ba133..030ce7a9 100644
--- a/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
+++ b/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
@@ -17,20 +17,13 @@ if(TARGET xnnpack OR xnnpack_POPULATED)
   return()
 endif()
 
-include(OverridableFetchContent)
+include(FetchContent)
 
-OverridableFetchContent_Declare(
-  xnnpack
-  GIT_REPOSITORY https://github.com/google/XNNPACK
-  # Sync with tensorflow/workspace2.bzl
-  GIT_TAG 659147817805d17c7be2d60bd7bbca7e780f9c82
-  GIT_PROGRESS TRUE
-  PREFIX "${CMAKE_BINARY_DIR}"
-  SOURCE_DIR "${CMAKE_BINARY_DIR}/xnnpack"
-)
-OverridableFetchContent_GetProperties(xnnpack)
+FetchContent_Declare(xnnpack SOURCE_DIR "${XNNPACK_SOURCE_DIR2}")
+
+FetchContent_GetProperties(xnnpack)
 if(NOT xnnpack_POPULATED)
-  OverridableFetchContent_Populate(xnnpack)
+  FetchContent_Populate(xnnpack)
 endif()
 
 # May consider setting XNNPACK_USE_SYSTEM_LIBS if we want to control all
