diff --git a/tensorflow/lite/tools/cmake/modules/gemmlowp.cmake b/tensorflow/lite/tools/cmake/modules/gemmlowp.cmake
index 0aaea326..5a77005f 100644
--- a/tensorflow/lite/tools/cmake/modules/gemmlowp.cmake
+++ b/tensorflow/lite/tools/cmake/modules/gemmlowp.cmake
@@ -17,25 +17,13 @@ if(TARGET gemmlowp OR gemmlowp_POPULATED)
   return()
 endif()
 
-include(OverridableFetchContent)
+include(FetchContent)
 
-OverridableFetchContent_Declare(
-  gemmlowp
-  GIT_REPOSITORY https://github.com/google/gemmlowp
-  # Sync with tensorflow/third_party/gemmlowp/workspace.bzl
-  GIT_TAG fda83bdc38b118cc6b56753bd540caa49e570745
-  # It's not currently (cmake 3.17) possible to shallow clone with a GIT TAG
-  # as cmake attempts to git checkout the commit hash after the clone
-  # which doesn't work as it's a shallow clone hence a different commit hash.
-  # https://gitlab.kitware.com/cmake/cmake/-/issues/17770
-  # GIT_SHALLOW TRUE
-  GIT_PROGRESS TRUE
-  SOURCE_DIR "${CMAKE_BINARY_DIR}/gemmlowp"
-)
+FetchContent_Declare(gemmlowp SOURCE_DIR "${gemmlowp_SOURCE_DIR2}")
 
-OverridableFetchContent_GetProperties(gemmlowp)
+FetchContent_GetProperties(gemmlowp)
 if(NOT gemmlowp_POPULATED)
-  OverridableFetchContent_Populate(gemmlowp)
+  FetchContent_Populate(gemmlowp)
 endif()
 
 set(GEMMLOWP_SOURCE_DIR "${gemmlowp_SOURCE_DIR}" CACHE PATH "Source directory")
